libraries/provision/ansible/playbooks/files/telegraf-sync-gateway.conf

```diff
@@ -0,0 +1,1577 @@
+# Telegraf Configuration
```

Comment:
For the product specific configs the redundant plugin config has been removed, this significantly improves readability.

Pushed to the new rebased branch.